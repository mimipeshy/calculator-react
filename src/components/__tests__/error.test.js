import React from 'react';
import renderer from 'react-test-renderer';
import Error from '../error';

describe('Error', () => {
  it('Tests if component renders correctly', () => {
    const tree = renderer
      .create(<Error />)
      .toJSON();
    expect(tree).toMatchSnapshot();
  });
});
