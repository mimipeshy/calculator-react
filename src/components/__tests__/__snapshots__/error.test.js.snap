// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Error Tests if component renders correctly 1`] = `
<p>
  Page not found!
</p>
`;
